db/migrate/20200430002042_create_work_experiences.rb

```diff
@@ -0,0 +1,15 @@
+class CreateWorkExperiences < ActiveRecord::Migration[5.2]
+  def change
+    create_table :work_experiences do |t|
+      t.references :student, foreign_key: true, null: false
+      t.references :company, foreign_key: true, null: false
```

Comment:
Not all companies will be stored on our side. I think that it would be better to make this a string.